pyvista/plotting/_property.py

```diff
@@ -398,6 +408,27 @@ def opacity(self, value: float):  # numpydoc ignore=GL08
         _check_range(value, (0, 1), 'opacity')
         self.SetOpacity(value)
 
+    @property
+    def edge_opacity(self) -> float:  # numpydoc ignore=RT01
+        """Return or set the edge opacity of this property.
+
+        Edge opacity of the mesh. A single float value that will be applied globally
+        edge opacity of the mesh and uniformly applied everywhere. Between 0 and 1.
+
+        """
+        if vtk_version_info < (9, 3):
+            return 1.0
+        else:  # pragma: no cover
```

Comment:
vtk 9.3.0 is released.  I also think we overuse these `no cover`s.  It would be easier to find places to add coverage or find blind spots in the code without them.

```suggestion
        else:
```